ui/app/components/app/connected-accounts-list/index.scss

```
@@ -0,0 +1,161 @@
+.connected-accounts-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__identicon {
+    margin-right: 8px;
+  }
+
+  &__account-name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__primary {
+    font-style: italic;
+  }
+
+  &__account-status {
+    color: $Grey-500;
+  }
+
+  &__account-status-link {
+    &, &:hover {
+      color: $curious-blue;
+      cursor: pointer;
+    }
+  }
+
+  &__primary, &__account-status {
+    font-size: 12px;
+    line-height: 17px;
+  }
+
+  &__row {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding: 16px 24px;
+
+    border-top: 1px solid $geyser;
+
+    &:last-of-type {
+      border-bottom: 1px solid $geyser;
+    }
+
+    &--highlight {
+      background-color: $warning-light-yellow;
+      border: 1px solid $warning-yellow;
+
+      box-sizing: border-box;
+      padding: 16px;
+      margin-bottom: 16px;
+      width: calc(100% - 16px)
+    }
+  }
+
+  &__row-content {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+}
+
+.connected-accounts-options-button {
+  background: none;
+}
+
+.connected-accounts-options {
+  background: $white;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.214);
+  border-radius: 8px;
+  min-width: 200px;
+  color: #24292E;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 16px;
+  right: 24px;
+
+  font-family: Roboto, 'sans-serif';
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 20px;
+
+  &__row {
+    background: none;
+    font-family: inherit;
+    font-size: inherit;
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    padding: 16px 0;
+    cursor: pointer;
+
+    & + &:last-of-type {
+      border-top: 1px solid $geyser;
+    }
+
+    &-icon {
+      margin-right: 8px;
+    }
+  }
+}
+
+.connected-accounts-permissions {
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+
+  font-size: 12px;
+  line-height: 17px;
+  color: #6A737D;
+
+  strong {
+    font-weight: bold;
+  }
+
+  p + p {
+    padding-top: 8px;
+  }
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    font-size: 14px;
+    line-height: 20px;
+    color: #24292E;
+
+    button {
+      font-size: 16px;
+      line-height: 24px;
+
+      background: none;
+      padding: 0 0 0 8px;
+    }
+  }
+
+  &__list {
+    padding-top: 8px;
+  }
+
+  &__list-item {
+    i {
+      padding-right: 8px;
```

Nit: this checkbox looks quite tiny. It was 18px in the designs.

```suggestion
      padding-right: 8px;
      font-size: 18px;
```

Though now it's not vertically centered with the text :thinking: